src/views/Contacts.vue

```diff
@@ -223,7 +229,8 @@ export default {
 
 		// generate groups menu from groups store
 		groupsMenu() {
-			return this.groups.map(group => {
+			this.$store.commit('sortGroups')
```

Comment:
You cannot have actions in a computed.
A computed is not allowed to change data on access.